Workbooks/RedisCache/AtResource/ResourceInsights.workbook

```diff
@@ -582,7 +589,7 @@
               ],
               "timeContextFromParameter": "TimeRange",
               "timeContext": {
-                "durationMs": 14400000
```

Comment:
Intentional? What does it mean to change this?

Reply:
Good question. I included the change for this `durationMs` to make the value consistent with the other graphs that included `"timeContextFromParameter": "TimeRange"`. In particular, I noticed that in the graphical editor for the workbook in the portal, when I changed `timeContextFromParameter` to `TimeRange`, it also saved a `0` for `durationMs`. My understanding is that when the time context is configured to read from a parameter, it overrides the `durationMs` property, so that property can just be `0` for simplicity.